File: firstcode/firstassignemt/newassignement/examresult.py
passes = 0
failure = 0

user_input = 0;
print(type(user_input))

while not user_input == -1:
	user_input = int(input("enter result (1 for pass:  2 for failure): "))
	if(user_input == 1):
		passes += 1
	elif not user_input ==  -1:
		failure += 1
print(f"the number of passes is {passes} and the number failures is {failure}")

File: firstcode/firstassignemt/myjavaassignment/ExamResult.java
import java.util.Scanner;
public class ExamResult {
	public static void main(String[] args) {
		System.out.println("welcome: lets check the results");
	
	Scanner input = new Scanner(System.in);
		
	int passes = 0;
	int failures = 0;
	
	int userInput = 0;
	
	while(userInput != -1) {
	
	System.out.println("enter result (1 for pass:  2 for failure: ");
		userInput = input.nextInt();
		if(userInput == 1) {
		passes = passes + 1;
		} else if(userInput == 2 ) {
		failures = failures + 1;
		}
}
		
	System.out.printf("the number of passes is %d and the number of failure is %d%n", passes, failures);	
}
}